dm/master/server.go

```diff
@@ -450,29 +447,46 @@ func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.S
 		for _, stCfg := range stCfgs {
 			sources = append(sources, stCfg.SourceID)
 		}
-		s.removeMetaLock.Lock()
+
+		var (
+			latched = false
+			release scheduler.ReleaseFunc
+			err3    error
+		)
+
 		if req.RemoveMeta {
+			// use same latch for remove-meta and start-task
+			release, err3 = s.scheduler.AcquireSubtaskLatch(cfg.Name)
```

Comment:
now rely on manually retry on user side, to try to quickly respond 😂

Reply:
If we output an error with the message something like "conflict with other commend please retry ...", and the user need to do nothing but retry,  then is it better to automatically retry by the server. 🤔

Reply:
If we should let the server perform a retry, it is equal to we use a blocking lock (maybe with timeout) rather than fail-fast lock. I prefer implement `Acquire` in future PRs (now we only have `TryAcquire`)